app/v1/services/ListPeriodSummariesService.scala

```diff
@@ -38,16 +38,27 @@ class ListPeriodSummariesService @Inject() (connector: ListPeriodSummariesConnec
       logContext: EndpointLogContext,
       correlationId: String): Future[ServiceOutcome[ListPeriodSummariesResponse[PeriodDetails]]] = {
 
-    connector.listPeriodSummaries(request).map(_.leftMap(mapDownstreamErrors(desErrorMap)))
+    connector.listPeriodSummaries(request).map(_.leftMap(mapDownstreamErrors(errorMap)))
   }
 
-  private val desErrorMap =
-    Map(
-      "INVALID_NINO"            -> NinoFormatError,
-      "INVALID_INCOME_SOURCEID" -> BusinessIdFormatError,
-      "NOT_FOUND_INCOME_SOURCE" -> NotFoundError,
-      "SERVER_ERROR"            -> InternalError,
-      "SERVICE_UNAVAILABLE"     -> InternalError
+  private def errorMap = {
+    val errors =
+      Map(
+        "INVALID_NINO"            -> NinoFormatError,
+        "INVALID_INCOME_SOURCEID" -> BusinessIdFormatError,
+        "NOT_FOUND_INCOME_SOURCE" -> NotFoundError,
+        "SERVER_ERROR"            -> InternalError,
+        "SERVICE_UNAVAILABLE"     -> InternalError
+      )
+    val extraTysErrors = Map(
+      "INVALID_TAX_YEAR"            -> TaxYearFormatError,
+      "INVALID_INCOMESOURCE_ID"     -> BusinessIdFormatError,
+      "INVALID_CORRELATION_ID"      -> InternalError,
+      "RULE_TAX_YEAR_RANGE_INVALID" -> RuleTaxYearRangeInvalidError,
+      "FORMAT_TAX_YEAR"             -> TaxYearFormatError,
```

Comment:
These two aren't downstream errors. Will also need to remove from service spec and integration spec.
```suggestion
```